### ISSUE TITLE
ident protocol implementation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7254]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#233|
|Type|enhancement|
|Created|2003-08-26 08:52:52Z|

<details><summary>Searchable metadata</summary>

```
trac-id__233 233
type__enhancement enhancement
reporter__itamarst itamarst
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1061887972000000 1061887972000000
changetime__1467042696857255 1467042696857255
version__ 
owner__ 
cc__exarkun cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
See http://www.amk.ca/python/simple/identd.py.html for nice
sample code for server.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Kind of done.  Protocol implementation available, but no support for behaving 
like a truthful ident server.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Done in r13464
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 4b3754826dfeca054b524c899dba6e65a3bfe1df
```
#!CommitTicketReference repository="" revision="4b3754826dfeca054b524c899dba6e65a3bfe1df"
Merge pull request #233 from acabhishek942/8510-acabhishek942-doc

Improper epytext removed from twisted.words.protocols.jabber.sasl_mechanisms

Author: acabhishek942
Reviewer: ralphm
Fixes: #8510
```
